blog/2024/2024-06-17-valibot-a-new-approach-to-data-validation-in-javascript.md

```diff
@@ -18,17 +18,17 @@ If video is your jam, check out this highlight from the live stream that summari
 
 During [his thesis work](https://valibot.dev/thesis.pdf), developer Fabian Hiller found himself with dedicated time to pursue an idea he'd been mulling over - creating a new modular data validation library for JavaScript. This led to the birth of Valibot.
 
-Hiller had previously worked on [modular forms](https://modularforms.dev/), but he wanted to bring that same modular philosophy to data validation. While popular validation libraries like [Zod](https://zod.dev/) offer excellent APIs, Hiller felt there was room to take modularity even further.
+Fabian had previously worked on [Modular Forms](https://modularforms.dev/), but he wanted to bring that same modular philosophy to data validation. While popular validation libraries like [Zod](https://zod.dev/) offer excellent APIs, Fabian felt there was room to take modularity even further.
```

Comment:
These changes have already been made on the dev.to article, https://dev.to/opensauced/valibot-a-new-approach-to-data-validation-in-javascript-1mgb .